project.py

```diff
@@ -77,7 +78,8 @@ def action_create():
 
 def action_grant():
     if not ksclient.is_valid_user(email=options.user, domain=options.domain):
-        himutils.sys_error('User %s not found as a valid user.' % options.user)
+        log_error("WARNING: user %s not found as a valid user!" % options.user, 0)
+        himutils.sys_error('User %s not found as a valid user.' % options.user, 0)
```

Comment:
Hva er dette?

Reply:
tenkte bare at det ikke skulle exite så la til 0

Reply:
Det er ok med exit 1 der. Dersom brukeren ikke finnes gir ikke grant til den samme brukeren noen mening uansett.

Lurte mest på at det nå er to forskjellige funksjoner som blir kalt.